content/SCALECLIReference/Auth/CLIApiKey.md

```diff
@@ -10,6 +10,193 @@ tags:
+
+{{< expand "Viewing API Key Instances" "v" >}}
+
+#### Description
+The `get_instance` command requires the `id` property.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table of properties for the specified API key when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth api_key get_instance id=<i>number</i></code>
+
+Where *number* is the list number of the API key you want to delete. For example, the first API key created on the system would be "1".
+
+{{< expand "Command Example" "v" >}}
+```
+auth api_key get_instance id=2
++------------+---------------------------+
+|         id | 2                         |
+|       name | apikey2                   |
+| created_at | 2023-08-22T19:58:59+00:00 |
+|  allowlist | <list>                    |
++------------+---------------------------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Query Command
+
+The `query` command returns a table of properties for all API keys.
+
+{{< expand "Running an API Key Query" "v" >}}
+
+#### Description
+The `query` command has no requirements.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table of properties for all API keys when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+`auth api_key query`
+
+{{< expand "Command Example" "v" >}}
+```
+auth api_key query
++----+---------+---------------------------+-----------+
+| id | name    | created_at                | allowlist |
++----+---------+---------------------------+-----------+
+| 2  | apikey2 | 2023-08-22T19:58:59+00:00 | <list>    |
+| 3  | apikey3 | 2023-08-22T20:14:59+00:00 | <list>    |
+| 4  | apikey4 | 2023-08-22T20:17:44+00:00 | <list>    |
++----+---------+---------------------------+-----------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Update Command
+
+The update command allows you to update existing API keys.
+
+{{< expand "Updating API Keys" "v" >}}
+
+#### Description
+The `update` command has two configurable properties and one argument. After specifying the `id` of the API key you want to update, you must include at least one of the configurable properties or arguments.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table of properties for all API keys when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth api_key update id=<i>number</i> name=<i>name</i> allowlist=[{"method":"<i>METHOD</i>","resource":"<i>api.resource</i>"}] reset=<i>true/false</i></code>
+
+Where 
+* *number* is the list number of the API key you want to update. For example, the first API key created on the system would be "1".
+* *name* is the new name you want to assign to the key.
+* *METHOD* is the HTTP method you want the key to use. Options are GET, POST, PUT, DELETE, CALL, and SUBSCRIBE.
+* *api.resource* is the WebSocket API resource you want to use. Append "/api/docs/" to the end of your TrueNAS web UI address to see our full list of WebSocket API resources.
```

Comment:
The "append" sentence belongs in the Description section where you explain the properties. These appear to be the same as the create command so you only have to do this once, probably in a table, but you can use sentences to explain them, and the syntax for how to enter the property arguments for each.